### ISSUE TITLE
Filebeat template missing `fips_enabled` key

### ISSUE THREAD

**Issue by andresrc** [elastic/beats#22608]
Please post all questions and issues on https://discuss.elastic.co/c/beats
before opening a Github Issue. Your questions will reach a wider audience there,
and if we confirm that there is a bug, then you can open a new issue.

For security vulnerabilities please only send reports to security@elastic.co.
See https://www.elastic.co/community/security for more information.

Please include configurations and logs if available.

For confirmed bugs, please report:
- Version: 7.10.0
- Operating System: macOS / others 
- Discuss Forum URL:
- Steps to Reproduce: 

Execute `filebeat setup --pipelines --modules aws`

You'll get the error

```
Exiting: Error getting config for fileset aws/s3access: Error interpreting the template of the input: template: text:40:6: executing "text" at <.fips_enabled>: map has no entry for key "fips_enabled"
```

Seems introduced in either #21585 or #21626

**Comment by edwin-va** [comment]
I am facing the same issue in version 7.10

**Comment by jsoriano** [comment]
I think that the fix would be to add `fips_enabled` to the list of variables in the manifests of the filesets.

As a workaround in the meantime, the missing variable can be set with `-M 'aws.*.var.fips_enabled=false'`:
```
filebeat setup --pipelines --modules aws -M 'aws.*.var.fips_enabled=false'
```

**Comment by edwin-va** [comment]
Hello @jsoriano , I tried manually adding the missing variable by using the steps above. But I am still receiving the same error.

**Comment by edwin-va** [comment]
@jsoriano . I was using version 7.9 and updated it to version 7.10

Well the other workaround worked. I added fips_enabled on ecah manifest.yml file and it working perfect now.

Thanks a lot @jsoriano

**Comment by jsoriano** [comment]
Fix https://github.com/elastic/beats/pull/22645 will be included in 7.10.1, other versions are not affected.
